docs/output_url_field.md

```diff
@@ -0,0 +1,30 @@
+# Using the clusterlogforwarder.output.url field
+
+The `clusterlogforwarder.output.url` field requires a valid absolute URL.  An
+'absolute' URL is one with non-empty `scheme` and `host:port` parts, in other
+words it must start with "something://hostname". User and password parts
+(e.g. http://user:pass@hostname) are *not allowed*, credentials should be
+provided in the `output.secret` field.
+
+In some cases an output type may provide an alternative way to configure
+connections, e.g. the `output.kafka.brokers` field. In such cases the output.url
+can be omitted. Using the output.url field is preferred whenever possible, but
+some output features may not be available via URL (e.g. kafka allows multiple
+failover addresses via `output.kafka.brokers`).
+
+Some output types define a URL scheme, for example elasticsearch uses `http` and
+`https`. Other output types (e.g. syslog) don't define a "natural" URL
+scheme. For those output types we use these special schemes:
+
+* tcp: insecure TCP connection.
```

Comment:
Do we really need or want to support insecure?

Reply:
I'm inclined to - developers who are testing forwarding to an external server will appreciate not having to screw around with setting up certificates in early development. Unlike our own elastic instance, we won't be setting that up for them. Also it seems likely that some users will rely on provider security and treat their cluster like a private network so again might not want to hassle with certs.

Reply:
We can review that it if you think it's against the opinionated policy. Secure is more likely in production, and almost nobody uses plain HTTP anymore.